templates/workspace_services/azureml/terraform/outputs.tf

```diff
@@ -28,6 +32,26 @@ data "azurerm_network_service_tags" "storage_tag" {
   location_filter = azurerm_storage_account.aml.location
 }
 
+data "azurerm_network_service_tags" "mcr_tag" {
```

Comment:
what is special about those kind of tags that we have to do this and not just use a name like we do elsewhere (like AzureActiveDirectory)?

Reply:
They are region tags to restrict to the region. Like `MicrosoftContainerRegistry.WestEurope`, this is dependant on the region being deployed to.